synfig-studio/src/gui/dialogs/canvasproperties.cpp

```diff
@@ -128,7 +128,8 @@ CanvasProperties::CanvasProperties(Gtk::Window& parent,etl::handle<synfigapp::Ca
 	ok_button->signal_clicked().connect(sigc::mem_fun(*this, &studio::CanvasProperties::on_ok_pressed));
 
 	get_vbox()->show_all();
-	refresh();
+	//refresh();
```

Comment:
You can remove that commented code.

Reply:
Alright! Thanks for the advice. I was a bit hesitant to delete code, but I suppose that is what git is for. I've updated the PR.

Reply:
You can resolve this review using the Resolve button :)